grails-app/views/newConsentGroup/index.gsp

```diff
@@ -26,7 +26,8 @@
             serverURL: "${webRequest.baseUrl}",
             fillablePdfURL : "${createLink(controller: 'newConsentGroup', action: 'downloadFillablePDF', method: 'GET')}",
             projectKey: "${params.projectKey}",
-            loadingImage: "${resource(dir: 'images', file: 'loading-indicator.svg')}"
+            loadingImage: "${resource(dir: 'images', file: 'loading-indicator.svg')}",
+            projectType: '${params.type}'
```

Comment:
This is another case where I'm not sure this is the right approach. I think we need another way to pass the props into the component other than this.

I must have missed this before, but the user here should also be refactored. We should have a url that the component can call that queries for the session user - it should not be passed in as a constant.